### ISSUE TITLE
Problem with socket configuration 

### ISSUE THREAD

**Issue by sadiqhabib** [wachunga/omega#18]
Hi i am trying to use this project but i am getting this error

```
TypeError: Cannot read property 'prototype' of undefined
    at Object.<anonymous> (<project-path>\omega\node_modules\socket.io\lib\store.js:35:41)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (<project-path>\omega\node_modules\socket.io\lib\manager.js:14:13)
    at Module._compile (module.js:569:30)
```

**Comment by micsun-al** [comment]
+1

**Comment by sadiqhabib** [comment]
@micsun-al this project is using old version of socket.io. so the only solution is you have to downgrade your npm and node version, Or implement new version in this project :-)

**Comment by wachunga** [comment]
I would definitely accept a PR to use modern socket.io or even just plain websockets. It's on my list but I just haven't made the time.

**Comment by diginfo** [comment]
+1 - I have this issue too

**Comment by diginfo** [comment]
Seems to be working now, by commenting out that.io.configure which no longer exists in socket.io:

 **/lib/tracker.js**

```
tracker.init = function (app, projectDao, realIssueDao) {
	var that = this;
	that.io = sio.listen(app);
	
	/*
	that.io.configure(function () {
		// excluded websocket due to Chrome bug: https://github.com/LearnBoost/socket.io/issues/425
		that.io.set('transports', ['htmlfile', 'xhr-polling', 'jsonp-polling']);
	});
  */

	issueDao = realIssueDao;
	projectDao.findAll(function (err, projects) {
		_.each(projects, function (project) {
			that.listen(project);
		});
	});
};
```

**Comment by wachunga** [comment]
Published as 1.1.0.
